CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+ Please report issues here [Issues - fossasia/open-event-android](https://github.com/fossasia/open-event-android/issues)
+
+### Branch Policy
+
+#### Sending pull requests:
+
+Say you want to contribute changes to someone else’s repository (eg, this one).
+
+It is advisable to clone only the development branch using following command:
+Go to the repository on github. (This repository is by fossasia, and is called open-event-android, then you’ll find it at http://github.com/fossasia/open-event-android.)
+
+
+Click the “Fork” button at the top right.
+
+You’ll now have your own copy of that repository in your github account.
+
+Open a terminal/shell.
+
+Type
+
+`$ git clone git@github.com:username/open-event-android`
+
+where 'username' is your username.
+
+You’ll now have a local copy of your version of that repository.
+
+*For further details on how to contribute as a first time contributer visit [Contribute to someone's repository](http://kbroman.org/github_tutorial/pages/fork.html)*
+
+#### Change into that project directory (open-event-android):
+
+`$ cd open-event-android`
+
+#### Add a connection to the original owner’s repository.
+
+`$ git remote add foss_remote git://github.com/fossasia/open-event-android`
+
+Note the distinction between git@github.com: in the first case and git://github.com/ in the second case. I’m not sure why these need to be the way they are, but that’s what works for me.
+
+Also note 'foss_remote' does not need to be specific. You could very well choose:
+
+`$ git remote add repo_nickname git://github.com/fossasia/open-event-android`
+
+#### To check this remote add set up:
+
+`$ git remote -v`
+
+#### Make changes to files.
+
+`git add` and `git commit` those changes
+
+`git push` them back to github. These will go to your version of the repository.
+
+Note: if you get an error like:
+
+error: src refspec master does not match any.
+
+error: failed to push some refs to 'git@github.com:username/the_repo'
+
+Then try `git push origin HEAD:gh-pages` (see stackoverflow.). 
+
+Typing `git show-ref` can show what reference to put after HEAD.
+
+#### Now Create a PR (Pull Request)
+Go to your version of the repository on github.
+
+Click the “Pull Request” button at the top.
+
+Note that FOSSASIA’s repository will be on the left and your repository will be on the right.
+
+Click the green button “Create pull request”. Give a succinct and informative title, in the comment field give a short explanation of the changes and click the green button “Create pull request” again.
+
+#### Pulling others’ changes
+Before you make further changes to the repository, you should check that your version is up to date relative to FOSSASIA’s version.
+
+Go into the directory for the project and type:
+
+`$ git pull foss_remote master`
+
+This will pull down and merge all of the changes that have been made in the original FOSSASIA repository.
+
+Now push them back to your github repository.
+
+`$ git push`
```

Comment:
Make them push a single branch to single remote

Reply:
Thank you for the review! I am currently implementing these changes